components/automate-ui/src/app/page-components/profile/profile.component.html

```diff
@@ -54,6 +54,16 @@
               License <chef-icon aria-hidden="true" class="dropdown-link-icon">launch</chef-icon>
             </a>
           </li>
+          <li>
+            <a href="https://www.chef.io/patents" target="_blank"
```

Comment:
```suggestion
            <a href="https://www.chef.io/patents" target="_blank" rel="noopener"
```

Suggestion for added security: https://web.dev/external-anchors-use-rel-noopener/

The summary: 
```
When you link to a page on another site using the target="_blank" attribute, you can expose your site to performance and security issues:

The other page may run on the same process as your page. If the other page is running a lot of JavaScript, your page's performance may suffer.
The other page can access your window object with the window.opener property. This may allow the other page to redirect your page to a malicious URL.
Adding rel="noopener" or rel="noreferrer" to your target="_blank" links avoids these issues.
```